.travis.yml

```diff
@@ -6,10 +6,6 @@ services:
 language: java
 jdk: oraclejdk11
 
-env:
-  global:
-  - IMAGE_NAME="$DOCKER_GCP_LOCATION/census-rm-action-scheduler"
```

Comment:
I'm not sure that removing this is really an improvement? Was it left in by mistake?

Reply:
In previous PR this variable was removed as the image location is not deemed to be sensitive information. The image name variable is unused so this is safe to delete.